index.html

```diff
@@ -101,127 +92,136 @@ <h2 class="header-text">My Recent Works</h2>
-            </div>
-            <!-- sixth -->
-            <div class="work-card">
-                <h2 class="card-header">Professional Art <br> Printing Data</h2>
-                <p class="card-description">
-                    A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
-                </p>
-                <div>
-                    <ul class="card-tags">
-                        <li><button class="tag blur">html</button></li>
-                        <li><button class="tag blur">bootstrap</button></li>
-                        <li><button class="tag blur">Ruby</button></li>
-                    </ul>
+                <!-- Fifth -->
+                <div class="work-card">
+                    <h2 class="card-header">Professional Art <br> Printing Data</h2>
+                    <p class="card-description">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                    </p>
+                    <div>
+                        <ul class="card-tags">
+                            <li><button class="tag blur">html</button></li>
+                            <li><button class="tag blur">bootstrap</button></li>
+                            <li><button class="tag blur">Ruby</button></li>
+                        </ul>
+                    </div>
+                    <button class="btn-max-width hide-btn">See Project</button>
+                </div>
+                <!-- sixth -->
+                <div class="work-card">
+                    <h2 class="card-header">Professional Art <br> Printing Data</h2>
+                    <p class="card-description">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                    </p>
+                    <div>
+                        <ul class="card-tags">
+                            <li><button class="tag blur">html</button></li>
+                            <li><button class="tag blur">bootstrap</button></li>
+                            <li><button class="tag blur">Ruby</button></li>
+                        </ul>
+                    </div>
+                    <button class="btn-max-width hide-btn">See Project</button>
                 </div>
-                <button class="btn-max-width">See Project</button>
             </div>
         </div>
     </div>
     </section>
     <section class="about-section">
-        <div>
-            <h1  class="primary-text">about me</h1>
-        </div>
-        <div>
-            <h2 class="description">
-            Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-            </h2>
+        <div class="about-container">
```

Comment:
- There is extra space on the right side of this section. Please use the full width to display your background images. 
- Please kindly add more padding on the button to match it with the  figma template  

Figma Design
<img width="894" alt="Screenshot 2022-12-10 at 10 23 45 PM" src="https://user-images.githubusercontent.com/24832846/206867966-742e5153-b6ec-4733-aff3-6b834de90535.png">

Your design 
<img width="1440" alt="Screenshot 2022-12-10 at 11 08 48 PM" src="https://user-images.githubusercontent.com/24832846/206868046-a5e2183a-976a-43e0-9118-2d57be2edd24.png">

Reply:
- I have used full width for the background illustration
- I have added more padding to button to match Figma design

Reply:
> * There is extra space on the right side of this section. Please use the full width to display your background images.
> * Please kindly add more padding on the button to match it with the  figma template
> 
> Figma Design <img alt="Screenshot 2022-12-10 at 10 23 45 PM" width="894" src="https://user-images.githubusercontent.com/24832846/206867966-742e5153-b6ec-4733-aff3-6b834de90535.png">
> 
> Your design <img alt="Screenshot 2022-12-10 at 11 08 48 PM" width="1440" src="https://user-images.githubusercontent.com/24832846/206868046-a5e2183a-976a-43e0-9118-2d57be2edd24.png">

Just as the previous reviewer noted in this quote, the issue still persists and it would be better to use width: 100% so that it will take the full space.
### Your project
![image](https://user-images.githubusercontent.com/86200356/206875427-e7dd18df-2b4c-4728-addb-c1b6a4e06c76.png)